.eslintrc.js

```diff
@@ -81,6 +81,7 @@ module.exports = {
 			rules: {
 				...defaultRules,
 				'vue/multi-word-component-names': 'off',
+				'vue/require-default-prop': 'off',
```

Comment:
Generally, I'm in favor of this. Just wondering if there are any downsides that might come with this as it is a strongly recommended rule: https://eslint.vuejs.org/rules/require-default-prop.html

Reply:
Hannes and I had the same doubt. I think typescript makes this rule redundant. The goal of the rule is to make sure you can reliably use a prop with the expected type even if it's omitted (so you don't accidentally try to use undefined). Typescript already handles that type safety!

Reply:
I think you're right 😃👍